app/actions/sessions.js

```diff
@@ -333,14 +360,14 @@ export function setConnectionNeedToBeSaved(tabId, bool) {
 function PREVIEW_QUERY (dialect, table, database = '') {
     switch (dialect) {
         case DIALECTS.IBM_DB2:
-            return `SELECT * FROM ${table} FETCH FIRST 5 ROWS ONLY`;
+            return `SELECT * FROM ${table}`;
         case DIALECTS.APACHE_SPARK:
         case DIALECTS.MYSQL:
         case DIALECTS.SQLITE:
         case DIALECTS.MARIADB:
         case DIALECTS.POSTGRES:
         case DIALECTS.REDSHIFT:
-            return `SELECT * FROM ${table} LIMIT 5`;
+            return `SELECT * FROM ${table}`;
```

Comment:
why was the `limit 5` removed? if there are a gazillion rows, this will probably break the app. Instead of `limit 5`, could we just do like `limit 1000`?

Reply:
in either case, the two dialects below this one should get updated as well

Reply:
I guess it was done because some charts with only 5 points don't look good.

Reply:
I'll set the limit to 1000.